### DISCUSSION TITLE
#help# how does trait count factor compute？？？

### DISCUSSION THREAD

**Discussion by paysage** [OpenRarity/open-rarity#98] [Q&A] [answered]
acording the ic algorithm，i got  the rank result  of azuki ，the first is #3309，but openrarity result is #2152，the #2152 token has unique trait count ： 5，Am I missing something？？？ How do I calculate this？？？？

**Comment by impreso** [comment, marked as answer]
Thanks for the feedback - could you pls tell what are you trying to do what ranks you compare? Maybe you should give us reference to TokenId in Azuki and how you obtain different ranks for tokens. THanks in Advance!

**Reply by paysage** [reply]
thx for resp,  i have solved this problem  in my go version rank server(implement openrarity  algorithm),  the rank result is same with OpenRarity now，this is the unique_trait_count compute way : https://github.com/OpenRarity/open-rarity/pull/67.
